app/views/components/forms/question_types/_text_phone_field.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="radios" role="group" aria-labelledby="<%= question.answer_field %>">
+  <%= render 'components/question_title', question: question %>
+  <% if question.character_limit && question.character_limit > 0 %>
+    <%= text_field_tag question.answer_field.to_sym, nil, type: :email, class: "usa-input", required: question.is_required, maxlength: question.max_length, onkeyup: "touchpointForm#{form.short_uuid}.textCounter(this,#{question.max_length});" %>
+	<span class="counter-msg usa-hint usa-character-count__message  aria-live="polite">
+		<%= question.max_length %> characters allowed
+	</span>
+  <% else %>
+    <%= telephone_field_tag question.answer_field.to_sym, nil, class: "usa-input", required: question.is_required, maxlength: question.max_length %>
```

Comment:
this won't need a `maxlength`

Reply:
consider adding a pattern for frontend validation

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/tel